cmd/pg/wal_push.go

```diff
@@ -35,7 +36,11 @@ var walPushCmd = &cobra.Command{
 			uploader.PGArchiveStatusManager = asm.NewNopASM()
 		}
 
-		postgres.HandleWALPush(uploader, args[0])
+		uploader.UploadingFolder = uploader.UploadingFolder.GetSubFolder(utility.WalPath)
+		err = postgres.HandleWALPush(uploader, args[0])
+		if err != nil {
```

Comment:
You can remove this if and leave only the `			tracelog.ErrorLogger.FatalOnError(err)` part.